packages/sns/src/swap.canister.ts

```diff
@@ -45,6 +46,12 @@ export class SnsSwapCanister extends Canister<SnsSwapService> {
   state = (params: QueryParams): Promise<GetStateResponse> =>
     this.caller(params).get_state({});
 
+  /**
+   * Notify of the payment failure to remove the ticket
+   */
+  notifyPaymentFailure = async (): Promise<NotifyPaymentFailureResponse> =>
```

Comment:
Why not provide the ticket that's inside `NotifyPaymentFailureResponse`

Reply:
I lik the idea, NotifyPaymentFailureResponse was removed